pandas/core/generic.py

```diff
@@ -3660,7 +3660,11 @@ def head(self, n=5):
 
     def tail(self, n=5):
         """
-        Return the last n rows.
+        Return the last `n` rows.
+
+        This function returns last `n` rows from the object based on the
```

Comment:
this is not fully correct. it returns the last n rows by position, regardless of the index.

Reply:
Yeah, I'll put in a PR for that one